src/run.js

```diff
@@ -36,13 +37,14 @@ const minimalcss = async options => {
       if (/data:image\//.test(request.url)) {
         // don't need to download those
         request.abort()
-      } else if (/\.(png|jpg|jpeg$)/.test(request.url)) {
+      } else if (/\.(png|jpg|jpeg|webp$)/.test(request.url)) {
```

Comment:
We probably want to skip `.gif` too. And I bet there are a bunch others we've forgotten too.

Reply:
well let's start with something and will see. I will add gif